main.js

```diff
@@ -0,0 +1,9 @@
+const navToggle = () => {
+  const burger = document.querySelector('.toggle');
+  const toggler = document.querySelector('.toggle');
```

Comment:
- [ ] Nice job 👏  on making the mobile menu functionality I can see everything is working 👏  however to apply javascript best practice it should be good to remove repeated code like here 👆  burger = toggle 
 Your code 
```js
  const navToggle = () => {
       const burger = document.querySelector('.toggle');
       const toggler = document.querySelector('.toggle');
       burger.addEventListener('click', () => {
          toggler.classList.toggle('active');
    });
};
```

 A better code 
```js
  const navToggle = () => {
       const toggler = document.querySelector('.toggle');
       toggler.addEventListener('click', () => {
          toggler.classList.toggle('active');
    });
};
```